src/pocketmine/Player.php

```diff
@@ -1781,6 +1781,14 @@ public function exhaust(float $amount, int $cause = PlayerExhaustEvent::CAUSE_CU
 		return 0.0;
 	}
 
+	public function canConsumeObject(Consumable $consumable) : bool{
+		if($this->isCreative()){
+			return true;
+		}
+
+		return parent::canConsumeObject($consumable);
```

Comment:
This can be simplified to `return $this->isCreative() || parent::canConsumeObject($consumable);`